### PR TITLE
Dockerfile linting

### PR DESCRIPTION
**Pull request by ammarlakis**
**Description**

Changes proposed in this pull request:

- Fixing linting errors in Dockerfiles.

**Related issue(s)**

Split from #12964 
Related to #12854

### CODE REVIEW

File: tests/service-catalog/Dockerfile [hunk 1]
```diff
@@ -15,11 +15,13 @@ RUN CGO_ENABLED=0 go test -tags=acceptance -c ./test -o /servicecatalog.test
 FROM alpine:3.10
 RUN apk add --no-cache curl
 
+WORKDIR /app
+
 COPY --from=builder env-tester.bin /go/bin/env-tester.bin
 COPY --from=builder ./servicecatalog.test .
```
Comment (wozniakjan) [inline tests/service-catalog/Dockerfile:21]: do you know if it matters that `servicecatalog.test` moved inside of the container from `/servicecatalog.test` to `/app/servicecatalog.test`?

**Review by wozniakjan — APPROVED** [review]
/lgtm
/hold

feel free to unhold if the below mentioned comment doesn't cause any problems

File: tests/service-catalog/Dockerfile (hunk 1 above)
Comment (ammarlakis) [reply tests/service-catalog/Dockerfile:21]: no since `.` means current work directory, which is set to `/app`.

Comment (wozniakjan) [reply tests/service-catalog/Dockerfile:21]: I know, just wondering if some app expects it to be hardcoded path. Feel free to unhold at your convenience.

**Pull request merged**